### DISCUSSION TITLE
Can I insert Ads?

### DISCUSSION THREAD

**Discussion by FreeSoftwareServers** [outline/outline#5652] [General]
I'm wondering if there is a way to integrate ads into an outline site?

**Comment by tommoor** [comment]
Nope, it doesn’t make sense for the aims of this platform.

**Comment by FreeSoftwareServers** [comment]
While I appreciate that you didn't have the capacity to foresee that use case, I wish you'd consider it. What possible negative could come of it? 

Would it be hard to implement?

$$ makes the world go around...
